lib/classes/Variables.test.js

```diff
@@ -688,6 +688,23 @@ describe('Variables', () => {
       });
     });
 
+    it('should populate symlinks', () => {
+      const serverless = new Serverless();
+      const SUtils = new Utils();
+      const tmpDirPath = testUtils.getTmpDirPath();
+      const realFilePath = path.join(tmpDirPath, 'someFile');
+      const symlinkPath = path.join(tmpDirPath, 'refSomeFile');
+
+      SUtils.writeFileSync(realFilePath, 'hello world');
+      fse.ensureSymlinkSync(realFilePath, symlinkPath);
```

Comment:
We additionally should test if this works on Windows. I'm not sure what exactly `fse.ensureSymlinkSync` does there.

Reply:
Appveyor test has already passed, so this works fine on Windows as well.
https://ci.appveyor.com/project/serverless/serverless/build/1.0.682